src/main/java/net/sf/mzmine/parameters/parametertypes/filenames/FileNameParameter.java

```diff
@@ -99,7 +99,9 @@ public FileNameParameter cloneParameter() {
     public void setValueFromComponent(FileNameComponent component) {
         File compValue = component.getValue();
         if (extension != null) {
-            if (!compValue.getName().endsWith(extension))
+            if (!(compValue.getName().toUpperCase()
+                    .endsWith(extension.toUpperCase()) || compValue.getName()
+                    .toLowerCase().endsWith(extension.toLowerCase())))
```

Comment:
Yep I agree, this 3rd case is usually not covered in other applications (that was a digression of mine). Cheers ;)